.claude/docs/workflow/sonar-auto-fix.md

```
@@ -0,0 +1,77 @@
+2. **Issue 자동 생성** (`sonar-issue.yml`)
+   - `SONAR_TOKEN` repo secret 등록 (private 프로젝트만 필요)
+   - 첫 실행: `gh workflow run sonar-issue.yml`
+
+3. **PR 자동 생성** (`sonar-fix.yml`) — 기본은 dry-run
+   - `ANTHROPIC_API_KEY` secret 등록
+   - `SONAR_FIX_ACTIVE` repo variable 을 `true` 로 설정
+   - 그 전까지는 코멘트가 와도 dry-run notice 만 남기고 PR 안 만듦
+
+## 운영 루프
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)